frontend/src/components/projects-data/projects-data.component.html

```
@@ -0,0 +1,58 @@
+<div
```

HTMLHint (1.5.0)

[error] Doctype must be declared before any non-comment content.

(doctype-first)